healthcheck/client_test.go

```diff
@@ -0,0 +1,59 @@
+package healthcheck
+
+import (
+	"errors"
+	"net/http"
+	"testing"
+
+	"github.com/ONSdigital/go-ns/healthcheck/mock_healthcheck"
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+func TestUnitHealthcheckClient(t *testing.T) {
+
+	service := "myService"
+
+	Convey("Healthcheck returns nil when no error", t, func() {
```

Comment:
Consider using a given, when, then format when writing tests. It seems verbose at first, but makes for a nice consistent format - see an example here: https://github.com/ONSdigital/dp-dataset-exporter/blob/04a99a7d6fa0999f9e01ca815d707c2e1884df2a/filter/store_test.go#L32